README.md

```diff
@@ -57,113 +69,158 @@ Subdirectory README files should follow the same general format as this root fil
 
-### POST /auth
+A wallet that uses multiple "ledgers" (BTC has legacy addresses and segwit addresses that constitute different ledgers within CAP) would first query the endpoint using the ledger_id of the preferred ledger, and if that fails it can try other ledgers. If the sender was able to send to a segwit address, they would first query the BTC_SEGWIT id (1) and if that fails, they would try the BTC_LEGACY id (0).
 
-No Authorization (Public method)
+The "address_type"
 
-Gets a new JWT for the given user. The JWT can be used to authenticate with the other endpoints. JWT should last 5 minutes.
+<hr>
+
+## TIER 2 ENDPOINTS
+
+The following endpoints are required for a server to be CAP 2 compliant. These endpoints allow a wallet to know how to interact with a server on behalf of the user, in other words CREATE/UPDATE/DELETE address/user info.
+
+### POST /v1/address
+
+Authorization: Bearer {jwt}
+
+Adds or updates the address for the authenticated user of the given coin type.
 
 | Parameters | Description | Required | Sample Value |
 | ---------- | ----------- | -------- | ------------ |
-| username | The username of the user | Yes | "lane"
-| password | The password of the user | Yes | "SU93Rc00lpa55"
+| ledger_id | The ledger_id for this address | Yes | 1 (BTC_SEGWIT)
+| address | The user's address for this ledger | Yes | bc1qvw0ytfntx6zs0lfsruem6xwj0mewng523ktatp
 
-#### Example Response
+#### Example Usage
+
+* Alice recieves a payment from Bob and her wallet recognizes that a payment was recieved
+* Assuming Alice wants to update her address, her wallet will generate a new address
+* Alice's wallet uses her alias to construct a URL and HTTP Method
+* alice@ogdolo.com -> POST <https://opencap.ogdolo.com/address>
+* Alice's wallet authenticates with the CAP server using her credentials to get a JWT that will securely allow her wallet to update her address
+* Alice's wallet constructs the JSON request to send to the server
 
 ```javascript
-Code: 200
+Authorization: Bearer {jwt}
 {
-    "jwt": "fn9s8dfasp7fub2uifb928rbpfdsfgasd9f8sdfbsa9fbsdf8ushdf9wbf"
+    "ledger_id": 0,
+    "address": "bc1qvw0ytfntx6zs0lfsruem6xwj0mewng523ktatp",
 }
 ```
 
-## Optional Endpoints
+* Alice sends the request and her address is updated on the server
 
-The following endpoints are not REQUIRED to adhere to CAP, because most of those functions could also be handled by a webpage portal, or simply on the server backend side if the user is running their own server. However, IF a server or wallet are going to implement the functionality of the following endpoints, they should do it in this manner. If for some reason this is impossible, please submit an issue/PR so we can fix the problem.
 
-### POST /user
+### POST /v1/auth
 
 No Authorization (Public method)
 
-Create a new user on the server.
+Gets a new JWT for the given user. The JWT can be used to authenticate with the other endpoints. JWT should last 5 minutes.

```

Comment:
Do we need JWT for this task? Why not use HTTP Basic Authentication for every call?

Reply:
JWT are nice and stateless. I think they are preferred by most web devs these days

Reply:
Mmh, what's the point of requesting a JWT for 5 minutes? I don't think that it is worth the trouble as it's unlikely to do more than one request every 5 minutes.

Reply:
Exactly, use the JWT for the one or two request you are going to make then be done. thats how web apps typically work these days.

Reply:
I would not use JWT just because it's fancy tech. Basic Auth does the job as well and has great support as every http library can work with it.